docker/Dockerfile

```diff
@@ -34,7 +34,9 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     pkg-config \
     libopenblas-dev \
     postgresql-server-dev-all \
-    && rm -rf /var/lib/apt/lists/*
+    && rm -rf /var/lib/apt/lists/* \
```

Comment:
If gcc is only required in pg_ivm, why don’t we just remove it from the base builder and only install it in the pg_ivm builder instead?

Reply:
That's just the first error after I try to remove build-essential. I can do some try.

Reply:
That would be a much simpler solution, and more maintainable

Reply:
Agree. I meet more problem after changing to different combinations of GCC, Clang and Rust toolchain. C++ building is really a mess :smile_cat: Need more time for these issues.